include/zephyr/tracing/tracing.h

```diff
@@ -1655,100 +1655,6 @@
  */
 #define sys_port_trace_k_pipe_read_exit(pipe, ret)
 
-/**
- * @brief Trace Pipe cleanup entry
- * @param pipe Pipe object
- */
-#define sys_port_trace_k_pipe_cleanup_enter(pipe)
-
-/**
- * @brief Trace Pipe cleanup exit
- * @param pipe Pipe object
- * @param ret Return value
- */
-#define sys_port_trace_k_pipe_cleanup_exit(pipe, ret)
-
-/**
- * @brief Trace Pipe alloc init entry
- * @param pipe Pipe object
- */
-#define sys_port_trace_k_pipe_alloc_init_enter(pipe)
-
-/**
- * @brief Trace Pipe alloc init exit
- * @param pipe Pipe object
- * @param ret Return value
- */
-#define sys_port_trace_k_pipe_alloc_init_exit(pipe, ret)
-
-/**
- * @brief Trace Pipe flush entry
- * @param pipe Pipe object
- */
-#define sys_port_trace_k_pipe_flush_enter(pipe)
-
-/**
- * @brief Trace Pipe flush exit
- * @param pipe Pipe object
- */
-#define sys_port_trace_k_pipe_flush_exit(pipe)
-
-/**
- * @brief Trace Pipe buffer flush entry
- * @param pipe Pipe object
- */
-#define sys_port_trace_k_pipe_buffer_flush_enter(pipe)
-
-/**
- * @brief Trace Pipe buffer flush exit
- * @param pipe Pipe object
- */
-#define sys_port_trace_k_pipe_buffer_flush_exit(pipe)
-
-/**
- * @brief Trace Pipe put attempt entry
- * @param pipe Pipe object
- * @param timeout Timeout period
- */
-#define sys_port_trace_k_pipe_put_enter(pipe, timeout)
-
-/**
- * @brief Trace Pipe put attempt blocking
- * @param pipe Pipe object
- * @param timeout Timeout period
- */
-#define sys_port_trace_k_pipe_put_blocking(pipe, timeout)
-
-/**
- * @brief Trace Pipe put attempt outcome
- * @param pipe Pipe object
- * @param timeout Timeout period
- * @param ret Return value
- */
-#define sys_port_trace_k_pipe_put_exit(pipe, timeout, ret)
-
-/**
- * @brief Trace Pipe get attempt entry
- * @param pipe Pipe object
- * @param timeout Timeout period
- */
-#define sys_port_trace_k_pipe_get_enter(pipe, timeout)
```

Comment:
note that this still appears here:

doc/kernel/services/polling.rst

Reply:
> On another note, you don't happen to know what to do with: `subsys/tracing/sysview/SYSVIEW_Zephyr.txt` ?

needs to be changed as well, not sure how to support renaming of APIs in this context tbh